doc/material-migration.md

```diff
@@ -14,20 +14,23 @@ We have standardized the naming of styles and resource keys. This change aims to
 
 #### Changes to Resource Keys
 
-| Old Resource Key | New Resource Key |
-| ---- | ---- |
-| MaterialTextBoxClearGlyphWidth | TextBoxClearGlyphWidth |
-| MaterialTextBoxClearGlyphHeight | TextBoxClearGlyphHeight |
-| MaterialFilledTextBoxCornerRadius | FilledTextBoxCornerRadius |
-| MaterialFilledTextBoxPadding | FilledTextBoxPadding |
-| MaterialFilledTextBoxMinHeight | FilledTextBoxMinHeight |
-| MaterialFilledTextBoxBorderHeightFocused | FilledTextBoxBorderHeightFocused |
-| MaterialOutlinedTextBoxBorderThickness | OutlinedTextBoxBorderThickness |
-| MaterialOutlinedTextBoxCornerRadius | OutlinedTextBoxCornerRadius |
-| MaterialOutlinedTextBoxPadding | OutlinedTextBoxPadding |
-| MaterialOutlinedTextBoxMinHeight | OutlinedTextBoxMinHeight |
-| MaterialOutlinedTextBoxBorderThicknessFocused | OutlinedTextBoxBorderThicknessFocused |
-| MaterialOutlinedTextBoxBorderThicknessPointerOver | OutlinedTextBoxBorderThicknessPointerOver |
+| Old Resource Key                                  | New Resource Key                                |
+|---------------------------------------------------|-------------------------------------------------|
+| MaterialTextBoxClearGlyphWidth                    | TextBoxClearGlyphWidth                          |
+| MaterialTextBoxClearGlyphHeight                   | TextBoxClearGlyphHeight                         |
+| MaterialFilledTextBoxCornerRadius                 | FilledTextBoxCornerRadius                       |
+| MaterialFilledTextBoxPadding                      | FilledTextBoxPadding                            |
+| MaterialFilledTextBoxMinHeight                    | FilledTextBoxMinHeight                          |
+| MaterialFilledTextBoxBorderHeightFocused          | FilledTextBoxBorderHeightFocused                |
+| MaterialOutlinedTextBoxBorderThickness            | OutlinedTextBoxBorderThickness                  |
+| MaterialOutlinedTextBoxCornerRadius               | OutlinedTextBoxCornerRadius                     |
+| MaterialOutlinedTextBoxPadding                    | OutlinedTextBoxPadding                          |
+| MaterialOutlinedTextBoxMinHeight                  | OutlinedTextBoxMinHeight                        |
+| MaterialOutlinedTextBoxBorderThicknessFocused     | OutlinedTextBoxBorderThicknessFocused           |
+| MaterialOutlinedTextBoxBorderThicknessPointerOver | OutlinedTextBoxBorderThicknessPointerOver       |
+| OutlinedPasswordBoxBorderPadding                  | ***REMOVED***                                   |
+| OutlinedPasswordBoxBorderPaddingPointerOver       | ***REMOVED***                                   |
+| OutlinedPasswordBoxBorderPaddingFocused           | ***REMOVED***                                   |
```

Comment:
Missing OutlinedPasswordBoxBorderThicknessPointerOver change @Xiaoy312 no?

Reply:
Do we consider `OutlinedPasswordBoxBorderThicknessPointerOver` value change as a breaking change? it is already mentioned in here doc/styles/PasswordBox.md

Reply:
values change are non-breaking
plus we've made bigger change than throughout the year, and there was absolute 0 mention neither